pipenv/routines/graph.py

```diff
@@ -1,92 +1,59 @@
 import json as simplejson
-import os
 import sys
 from pathlib import Path
 
 from pipenv import exceptions
+from pipenv.utils import console, err
 from pipenv.utils.processes import run_command
 from pipenv.utils.requirements import BAD_PACKAGES
-from pipenv.vendor import click
 
 
 def do_graph(project, bare=False, json=False, json_tree=False, reverse=False):
     import json as jsonlib
 
     from pipenv.vendor import pipdeptree
 
-    pipdeptree_path = os.path.dirname(pipdeptree.__file__.rstrip("cdo"))
+    pipdeptree_path = Path(pipdeptree.__file__).parent
     try:
         python_path = project.python()
     except AttributeError:
-        click.echo(
-            "{}: {}".format(
-                click.style("Warning", fg="red", bold=True),
-                "Unable to display currently-installed dependency graph information here. "
-                "Please run within a Pipenv project.",
-            ),
-            err=True,
+        err.print(
+            "[bold][red]Warning: Unable to display currently-installed dependency graph information here. "
+            "Please run within a Pipenv project.[/red][/bold]",
         )
         sys.exit(1)
     except RuntimeError:
         pass
-    else:
-        if os.name != "nt":  # bugfix #4388
-            python_path = Path(python_path).as_posix()
-            pipdeptree_path = Path(pipdeptree_path).as_posix()
 
-    if reverse and json:
```

Comment:
Neat -- the pipdeptree you vendored in @oz123 supports these flags now 🎉

Reply:
Yay!